### PR TITLE
Result on map

### PR DESCRIPTION
**Pull request by rakeshgunduka**

### CODE REVIEW

**Comment by rakeshgunduka** [comment]
@planemad

File: components/DetailedHome.jsx [hunk 1]
```diff
@@ -53,6 +54,25 @@ export default function DetailedHome({ state, district, type }) {
...
+    const locationResources = {
+        get All() {
+            return []
+                .concat(this.Oxygen)
+                .concat(this.Medicine)
+                .concat(this.Hospital)
+                .concat(this.Ambulance)
+                .concat(this.Helpline)
+                .concat(this.Vaccine)
+                .sort((record) => (isVerified(record.verificationStatus) ? -1 : 1));
```
Comment (shrihari689) [inline components/DetailedHome.jsx:66]:
```suggestion
                .sort((record) => (isVerified(record.verification_status) ? -1 : 1));
```

From v2 API, we use `verification_status` 📍 Refer: https://github.com/coronasafe/life/wiki/Life-Data-Structure

File: components/DetailedHome.jsx [hunk 2]
```diff
@@ -62,8 +82,7 @@ export default function DetailedHome({ state, district, type }) {
                 .concat(this.Ambulance)
                 .concat(this.Helpline)
                 .concat(this.Vaccine)
-                .sort(record => isVerified(record.verificationStatus) ? -1 : 1)
-                    
+                .sort((record) => (isVerified(record.verificationStatus) ? -1 : 1));
```
Comment (shrihari689) [inline components/DetailedHome.jsx:85]: Ooops! 😁 Modify here too 📍

**Review by shrihari689 — CHANGES_REQUESTED** [review]

File: components/DetailedHome.jsx (hunk 2 above)
Comment (rakeshgunduka) [reply components/DetailedHome.jsx]: Updated

File: components/DetailedHome.jsx (hunk 1 above)
Comment (rakeshgunduka) [reply components/DetailedHome.jsx]: Renamed it to verification_status

**Comment by shrihari689** [comment]
Hurraay 🎉 Thanks @rakeshgunduka 📍

**Pull request merged**